modules/ROOT/pages/dataweave-pattern-matching.adoc

```diff
@@ -103,36 +106,31 @@ matching.
 
 Matches when the evaluated value equals a simple literal value.
 
-In this example, the first field matches the value in `payload.string` and
-returns a Boolean. The second field performs the same match, but returns an
-object that contains both a boolean and a reference to the validated value.
+In this example, the first field matches the value in `myInput.string` and
+returns a Boolean. The second field performs the same match, but it returns an
+object that contains both a Boolean and a reference to the validated value.
```

Comment:
Should be *boolean* in lower case?

Reply:
I've always capitalized it but admit it's weird not to capitalize String, too. Looks like it's usually capitalized: https://www.merriam-webster.com/dictionary/Boolean  Perhaps this practice is because the name comes from George Boole, the English mathematician...

Reply:
I'll leave this an open question.

Reply:
based on additional input from the team and the Oracle glossary, converting to lower case. Thx for the suggestion, Kevin! There will likely be inconsistencies elsewhere.